pkg/cosmos/client.go

```diff
@@ -156,6 +159,30 @@ func (c *Client) PackageRender(appID string, name string, version string, option
 	return render.MarathonJson, nil
 }
 
+// PackageInstall installs package
+func (c *Client) PackageInstalls(appID string, name string, version string, optionsPath string) error {
```

Comment:
I'm not sure I understand why this is called `PackageInstalls` vs `PackageInstall`

Reply:
Fixed